hazelcast/vector.py

```diff
@@ -65,20 +97,35 @@ class Metric(enum.IntEnum):
 
 @dataclasses.dataclass
 class IndexConfig:
+    """The IndexConfig contains configuration for a vector index.
+
+    Args:
+        name: Name of the vector index.
+        metric: The metric to be used for the index configuration.
+        dimension: The dimension of vectors to be used in the vector index.
+            All vectors that use this index configuration must have
+            the same dimension.
+        max_degree: The maximum number of connections allowed per node.
+        ef_construction: The size of the dynamic list for search.
+    """
 
     name: str
     metric: Metric
     dimension: int
-    max_degree: int = 200
-    ef_construction: int = 300
-    use_deduplication: bool = False
+    max_degree: int = 16
+    ef_construction: int = 100
+    use_deduplication: bool = True
```

Comment:
this flag is missing from the args docs above, let's add it?

Reply:
Added at: https://github.com/hazelcast/hazelcast-python-client/pull/678/commits/86856148d6d318eb453ee672625196cf2ad11e8c